vscode/src/client/analyzerClient.ts

```diff
@@ -694,7 +693,7 @@ export class AnalyzerClient {
     return true;
   }
 
-  public getAnalyzerPath(): string {
+  protected getAnalyzerPath(): string {
```

Comment:
For my edification. Why `public` -> `protected`?

Reply:
Just being pedantic.  `protected`  because it isn't intended to be called outside of the class (it's a helper function).  So all the public functions are ok to call from the other extension code.